docs/excel/custom-functions-authentication.md

```diff
@@ -0,0 +1,50 @@
+---
+ms.date: 1/28/2019
+description: Learn how to authenticate users via custom functions in Excel.
+title: Authentication for Custom Functions
+---
+
+# Authentication
+
+You may wish to verify that a user is authenticated before they access and use your custom function. While custom functions does not require a specific method of authentication, you should be aware that custom functions runs in a separate runtime from the task pane and other UI elements of your add-in. Because of this, you will need to pass data back and forth between the two runtimes using `AsyncStorage` and the Dialog API.
+  
+## AsyncStorage
+
+Custom functions' runtime does not have a `localStorage` object available on the global window, where you might typically store user information. Instead, custom functions and task panes both have access to another storage location, `OfficeRuntime.AsyncStorage`. It has a variety of [useful properties](https://docs.microsoft.com/en-us/javascript/api/office-runtime/officeruntime.asyncstorage?view=office-js) to set and store data. It essentially serves as a bridge between custom functions and your task pane (or other UI elements).
+
+Suggested usage:
+
+- Use `AsyncStorage` to set and store user tokens.
+- From your custom function, read that token from `AsyncStorage` and use it to make your web request.
```

Comment:
For consistency, say "From your custom function, read user tokens from 'AsyncStorage' and use them to make web requests.